fix(Block Storage): add warning in additional disk guide for luks type (#8403)
ovh/docs: pages/public_cloud/compute/create_and_configure_an_additional_disk_on_an_instance/guide.en-gb.md

>>
>> The types ending in -luks are encrypted (LUKS).
>>
>> > [!warning]
>> >

> >> > Creating a **-luks** volume automatically generates a dedicated key in Barbican.

Reviewer: not sure if the term barbican is well known by our customer ?
Maybe just saying Openstack is enough

>> >

> >> > Do not modify or delete this key if it is linked to one of your block volumes. Doing so will make the data on that volume permanently unrecoverable.

Reviewer 2: ```suggestion
>> > Do not modify or delete this key if it is linked to a block volume. Doing so will make the data on that volume and all its snapshots permanently unrecoverable.
```

Small suggestion that also mentions the snapshots
